requirements/extras.txt

```diff
@@ -7,3 +7,4 @@ numba
 numba-scipy
 pytest >= 5.1
 setuptools_scm
+tox
```

Comment:
As per the comment at the top of this file, the associated section in the `setup.cfg` also needs to be updated.

Reply:
Good thing we have that comment there so I don't forget to do things like that!  😸